docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.8'
+
+services:
+  chronicle-api:
+    container_name: chronicle-api
+    image: chronicle-api:1.0.0
+    build:
+      context: ./server
+      target: '${NODE_ENV}'
+      dockerfile: Dockerfile
+    command: npm run start:debug
+    ports:
+      - 3000:3000
+      - 9229:9229
+    networks:
+      - chronicle-network
+    volumes:
+      - ./server:/usr/src/app
+      - /usr/src/app/node_modules
+    restart: unless-stopped
+  chronicle-database:
+    image: mongo:latest
+    container_name: mongodb
+    volumes:
+      - mongodb_data_container:/data/db
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: user
+      MONGO_INITDB_ROOT_PASSWORD: password
```

Comment:
We should pass this using .env file: https://docs.docker.com/compose/environment-variables/#the-env-file

Reply:
That way we can synchronize it for both server and the database